performance/README.md

```diff
@@ -0,0 +1,42 @@
+Firebase Performance Quickstart
+==============================
+
+The Firebase Performance Android Quickstart app demonstrates measuring the performance of
+activities in your application.
```

Comment:
The description should go into a bit more detail:

- what is the quickstart is doing
- what exactly is perfmon measuring
- how can the dev verify that perfmon is working as expected

Reply:
Added some detail, lmk what you think